includes/class-sensei-quiz.php

```diff
@@ -1653,34 +1653,40 @@ public static function action_buttons() {
 		wp_enqueue_script( 'sensei-stop-double-submission' );
 		?>
 
-		<div class="wp-block-buttons">
+		<div id="sensei-quiz-actions">
 			<?php if ( $can_take_quiz ) : ?>
-				<div class="wp-block-button">
-					<button type="submit" name="quiz_complete" class="wp-block-button__link button quiz-submit complete sensei-stop-double-submission">
-						<?php esc_attr_e( 'Complete Quiz', 'sensei-lms' ); ?>
-					</button>
-
-					<input type="hidden" name="woothemes_sensei_complete_quiz_nonce" id="woothemes_sensei_complete_quiz_nonce" value="<?php echo esc_attr( wp_create_nonce( 'woothemes_sensei_complete_quiz_nonce' ) ); ?>" />
-				</div>
-
-				<div class="wp-block-button is-style-outline">
-					<button type="submit" name="quiz_save" class="wp-block-button__link button quiz-submit save sensei-stop-double-submission">
-						<?php esc_attr_e( 'Save Quiz', 'sensei-lms' ); ?>
-					</button>
-
-					<input type="hidden" name="woothemes_sensei_save_quiz_nonce" id="woothemes_sensei_save_quiz_nonce" value="<?php echo esc_attr( wp_create_nonce( 'woothemes_sensei_save_quiz_nonce' ) ); ?>" />
+				<div class="sensei-quiz-actions-primary wp-block-buttons">
+					<div class="sensei-quiz-action wp-block-button">
+						<button type="submit" name="quiz_complete" class="wp-block-button__link button quiz-submit complete sensei-stop-double-submission">
+							<?php esc_attr_e( 'Complete', 'sensei-lms' ); ?>
+						</button>
+
+						<input type="hidden" name="woothemes_sensei_complete_quiz_nonce" id="woothemes_sensei_complete_quiz_nonce" value="<?php echo esc_attr( wp_create_nonce( 'woothemes_sensei_complete_quiz_nonce' ) ); ?>" />
+					</div>
 				</div>
 			<?php endif ?>
 
-			<?php if ( $is_reset_allowed ) : ?>
-				<div class="wp-block-button is-style-outline">
-					<button type="submit" name="quiz_reset" class="wp-block-button__link button quiz-submit reset sensei-stop-double-submission">
-						<?php esc_attr_e( 'Reset Quiz', 'sensei-lms' ); ?>
-					</button>
-
-					<input type="hidden" name="woothemes_sensei_reset_quiz_nonce" id="woothemes_sensei_reset_quiz_nonce" value="<?php echo esc_attr( wp_create_nonce( 'woothemes_sensei_reset_quiz_nonce' ) ); ?>" />
-				</div>
-			<?php endif ?>
+			<div class="sensei-quiz-actions-secondary">
+				<?php if ( $is_reset_allowed ) : ?>
+					<div class="sensei-quiz-action">
+						<button type="submit" name="quiz_reset" class="sensei-stop-double-submission">
```

Comment:
I don't think we should drop all of those other CSS classes. People could be using them to style these buttons themselves. Probably the only one we could get rid of is `wp-block-button__link`. This will still break for people who are styling using this class, but probably they should have used one of the other classes. Same thing for Save.

Reply:
Makes sense for people to be able to target these buttons, but I guess we will break their custom style anyway with these changes. 😄 

Reverted the classes in bee960e8a4b9d16a565740c5b947a910d4798f30.